frontend/src/components/WorkflowSamplesTable/WorkflowSamplesTable.tsx

```diff
@@ -37,20 +38,20 @@ export interface WorkflowSamplesTableProps {
 	loading?: boolean
 }
 
-function WorkflowSamplesTable({ samples, columns, filterDefinitions, filterKeys, filters, setFilter, setFilterOptions, sortBy, setSortBy, pagination, selection, hasFilter, clearFilters, loading }: WorkflowSamplesTableProps) {
+function WorkflowSamplesTable({ samples, columns, filterDefinitions, filterKeys, filters, setFilter, setFilterOptions, sortByList, setSortByList, pagination, selection, hasFilter, clearFilters, loading }: WorkflowSamplesTableProps) {
 
-	const tableColumns = useMemo(() => {
-		const mergedColumns = addFiltersToColumns(
+	const columnsWithFilters = useMemo(() => {
+		return addFiltersToColumns(
 			columns,
 			filterDefinitions ?? {},
 			filterKeys ?? {},
 			filters ?? {},
 			setFilter,
-			setFilterOptions)
-		return mergedColumns
+			setFilterOptions
+		)
 	}, [columns, filterDefinitions, filterKeys, filters, setFilter, setFilterOptions])
-
-
+	// eslint-disable-next-line @typescript-eslint/no-empty-function
+	const tableColumns = useCustomMultiColumnSorter(columnsWithFilters, sortByList, setSortByList ?? (() => {}))
```

Comment:
You did not felt like typing () = {} as UndefinedCallback ?

Reply:
I could create a variable called `UndefinedCallback`.